packages/@react-spectrum/s2/src/TextField.tsx

```diff
@@ -197,6 +197,7 @@ function TextAreaInput() {
         color: '[inherit]',
         fontFamily: '[inherit]',
         fontSize: '[inherit]',
```

Comment:
is it going to be problematic for fontSize to be inherited, but lineHeight is 'ui' instead of inherit as well?

Reply:
hmm that's what it was before. I guess we could change it but not sure